cves/kernel/CVE-2019-12819.yml

```diff
@@ -448,7 +446,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: This was a lapse. They were not calling put_device() when device_register() fails.
 CWE_instructions: |
   Please go to http://cwe.mitre.org and find the most specific, appropriate CWE
   entry that describes your vulnerability. We recommend going to
```

Comment:
Same as up above, make sure to go back and manually confirm the CWE and change it to Manually confirmed.

Reply:
I missed this. Thanks, will do !